packit_service/worker/handlers/distgit.py

```diff
@@ -166,7 +171,13 @@ def sync_branch(self, branch: str) -> Optional[PullRequest]:
                     logger.info(f"Will retry for the {retries + 1}. time in {delay}s.")
                     # throw=False so that exception is not raised and task
                     # is not retried also automatically
-                    self.task.retry(exc=ex, countdown=delay, throw=False)
+                    self.task.retry(
+                        exc=ex,
+                        countdown=delay,
+                        args=(),
+                        # https://stackoverflow.com/questions/68915407/celery-retry-with-updated-arguments
+                        kwargs={"propose_downstream_run_id": model.id},
```

Comment:
This reusing is just for the purpose of the not-found-source, isn't it? (Just comparing this to the build workflow where we create a new model in case of an error. But if it is just for waiting, that's ok.)